### DISCUSSION TITLE
Broadcast Sample

### DISCUSSION THREAD

**Discussion by appelgriebsch** [WilliamVenner/searchlight#1] [Q&A]
Hey, I tried to run the sample code from `examples/broadcast.rs` on my M1 Mac, but its stuck on `Waiting for discovery...`. So looks like the discovery events are not being fired properly.

Any idea?

Update: tried to start with `RUST_LOG=debug` but didn't see any additional logs that may help investigating the issue....

**Comment by appelgriebsch** [comment]
Looks like on my Linux box it is working when I change the following two lines:

![image](https://user-images.githubusercontent.com/6803419/219120906-2e2a7455-7351-47f2-b685-c57f0375649a.png)

setting the `IpVersion` to `Both`

**Comment by WilliamVenner** [comment]
Are you possibly connected to a network that only has IPv6 available?
